pkg/utils/kube/crd.go

```diff
@@ -13,16 +13,16 @@ import (
 func CRDsInstalled(apiserverClient apiserver.Interface) error {
 	kyvernoCRDs := []string{
 		"admissionreports.kyverno.io",
-		"backgroundscanreports.kyverno.io",
-		"cleanuppolicies.kyverno.io",
+		// "backgroundscanreports.kyverno.io",
+		// "cleanuppolicies.kyverno.io",
 		"clusteradmissionreports.kyverno.io",
-		"clusterbackgroundscanreports.kyverno.io",
-		"clustercleanuppolicies.kyverno.io",
+		// "clusterbackgroundscanreports.kyverno.io",
+		// "clustercleanuppolicies.kyverno.io",
```

Comment:
Ideally these would be conditional based on something like a flag or environment variable that is set by Helm so that if you habe background scanning you get the CRDs from Helm and this will also ensure they are present.

Reply:
I am moving sanity checks in their respective controllers and check for their respective CRD presence. So incase we have enabled reportscontroller those CRDs will be checked while executing the code in reportscontroller `main.go`.

Reply:
Fixed